app/views/products/_account_products.html.erb

```diff
@@ -0,0 +1,56 @@
+<div class="col-md-9">
```

Comment:
From @lacuchilla:
Indentation gets slightly wonky in this file (and other view files)